drud/ddev
# NFS touchups to docs and debug command

## The Problem/Issue/Bug:

[Testing of v1.14.0-rc1](https://github.com/drud/ddev/issues/2174) found some docs and related issues that could be touched up, and this tries to do that.

## Manual Testing Instructions:

* Use `ddev debug nfsmount`.
* Go through the NFS instructions and debugging. See [new docs](https://github.com/drud/ddev/blob/c7a3672ca5b557ddfa783e16449654cd9647be59/docs/users/performance.md). It would be great to have somebody who has never set up NFS go through this.
